pegasus/sites.v3/code.org/public/css/values.scss

```diff
@@ -21,6 +21,7 @@
       text-align: center;
       display: flex;
       flex-direction: column;
+      align-items: center;
```

Comment:
Centered the icons in the values tiles on https://code.org/about/values:
<img width="500" alt="Screenshot 2023-05-18 at 4 05 41 PM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/d1732e3f-fc89-4707-b64b-4f2f40511ccc">

Reply:
Yep!